spec/struct.dd

```diff
@@ -566,6 +566,113 @@ $(GNAME Postblit):
         }
         ---
 
+        $(P There are situations in which a struct does not define a postblit,
+        but the compiler generates one: if one or more of the fields are themselves
+        structs that define postblits, then a field postblit is created that
+        calls all the underlying postblits. If a `struct` explicitly defines a
+        postblit operation, its generated name is `__postblit`. However, that is
+        just the code for `this(this)`, not including transitive calls to member
+        postblits. The generated name for the transitive postblit operation (fields
+        first in topological order) is `__xpostblit`. Neither function is defined
+        for `struct`s that don't define `this(this)` and don't have fields that
+        transitively define it. Example:)
+
+        ---
+        struct N1 { int a, b; }
+        struct N2 { N1 member; string s; }
+        struct S { this(this) {} }
+        struct B { S a; }
+
+        void main()
+        {
+            // No postblit operations for N1 or N2
+            static assert(!(__traits(hasMember, N1, "__xpostblit")));
+            static assert(!(__traits(hasMember, N1, "__postblit")));
+            static assert(!(__traits(hasMember, N2, "__xpostblit")));
+            static assert(!(__traits(hasMember, N2, "__postblit")));
+
+            // S has a defined postblit and a generated one
+            static assert((__traits(hasMember, S, "__xpostblit")));
+            static assert((__traits(hasMember, S, "__postblit")));
+
+            // B does not have a defined postblit, but has a generated one
+            static assert(!(__traits(hasMember, B, "__postblit")));
+            static assert((__traits(hasMember, B, "__xpostblit")));
+        }
+        ---
+
+        $(P Postblits are not overloadable.)
+        
+        $(P If a postblit is qualified with `immutable` or `shared` as in
+        $(D this(this) immutable;) or $(D this(this) shared;), then the code
+        is ill-formed. Historically, such definitions passed compilation but
+        could not be invoked. A deprecation diagnostic is issued by the front-end
+        of dmd 2.080 or later, see $(BUGZILLA 18417). Example:)
```

Comment:
Don't add things that can get outdated into the spec, because they will.
Info this belongs into the changelog entry